### DISCUSSION TITLE
Prerender just start page

### DISCUSSION THREAD

**Discussion by maxcodefaster** [scullyio/scully#1553] [Q&A] [answered]
Hello, is it possible to just prerender one page, for example the home page for logged out users, as sort of a landingpage, and leave the other dynamic pages as is?

**Comment by A7xSV** [comment]
Routes with dynamic params will not be pre-rendered by Scully by default. These are routes in your app router module that have a `/:dynamicParam`. All others will be pre-rendered by default. If you want to pre-render a route with a dynamic param you need to specify how which params to render with a router plugin (see https://scully.io/docs/Reference/plugins/types/router/).

If you want to specifically ignore certain routes, you can use the `ignored` plugin (https://scully.io/docs/faq/#ignoring-routes).

Useful resources to read about this topic:
- https://scully.io/docs/concepts/unhandled-routes/
- https://scully.io/docs/concepts/handled-routes/

**Comment by SanderElias** [comment, marked as answer]
Yes, you can. 
We have an `ignored` plugin, that allows you to 'skip' routes from the Scully generation.
Also, there is a way to set the default to `ignored` so it skips over anything that is not specified.
There is one exception. The top `/` route is always rendered.
